extras/package.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
question about this file - why do we need it?

Reply:
It allows us to do `cd examples && npm link ../extras`